src/shared/common/forms/radio-group.jsx

```diff
@@ -103,6 +103,10 @@ class Radio extends Component {
         return choicesToUse;
     }
 
+    getIdName(nameOfField, key) {
+        return `${nameOfField}-${key}`;
+    }
```

Comment:
dunno if this is a good idea, but:
```suggestion
    getIdName(nameOfField, key) => `${nameOfField}-${key}`;
```

Reply:
This is following the existing pattern - doesn't make any real difference.